sockeye/arguments.py

```diff
@@ -820,10 +838,24 @@ def add_training_args(params):
                               help='How to normalize the loss. By default loss is normalized by the number '
                                    'of valid (non-PAD) tokens (%s).' % C.LOSS_NORM_VALID)
 
+    train_params.add_argument('--length-ratio-task-weight',
+                              type=float_greater_or_equal(0.0),
+                              default=0.0,
+                              help='Weight for the loss part that corresponds to learning length ratios. Not used if equal to 0.0. Default %(default)s.')
+    train_params.add_argument('--length-ratio-task-layers',
+                              type=int_greater_or_equal(1),
+                              default=1,
+                              help='Number of fully-connected layers for predicting the length ratio. Default %(default)s.')
+    train_params.add_argument('--length-ratio-task-type',
```

Comment:
i would rename all flags to `--length-task*` since its not always ratio

Reply:
Done